xbmc/cores/VideoPlayer/VideoPlayerAudio.cpp

```diff
@@ -503,6 +503,9 @@ void CVideoPlayerAudio::Process()
         // guess next pts
         m_audioClock += audioframe.duration;
 
+		if (consumed >= pPacket->iSize) // quick fix for some decoder like wmalossless that cannot accept empty package
```

Comment:
apart from wrong indentation this is not correct either. decoder may produce more than a single output packet for a packet that goes in.
can you provide a sample that causes issues?